src/capwap_ac.erl

```diff
@@ -2613,3 +2618,32 @@ handle_session_timer_ev({_, Level, _} = Ev, {Interval, _, _Opts} = Timer,
 		Data0
 	end,
     {keep_state, Data}.
+
+add_location(Name, Map) when is_binary(Name) ->
```

Comment:
This and next function causes are too similar, it make sense create a small helper function like `to_bin/1` which you can use in one place and not repeat the code

Reply:
Ok, I'll factor it out.